scripts/update_from_templates.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function cleanup() {
+    cd "$1"
+    local BUNDLE_UUID="$2"
+    
+    # Replace specific names with company.databricks.com, user@company.com, user_name
+    find . -type f -exec sed -i '' -E 's|e2[^[:space:]]*\.com|company.databricks.com|g' {} \;
+    find . -type f -exec sed -i '' -E 's|[A-Za-z0-9._%+-]+@databricks\.com|user@company.com|g' {} \;
+    find . -type f -exec sed -i '' -e "s|$CURRENT_USER_NAME|user_name|g" {} \;    
+    find . -type f -exec sed -i '' -E "s|^([[:space:]]*uuid:[[:space:]]*)[^[:space:]]*[[:space:]]*$|\\1$BUNDLE_UUID|g" {} \;
+    
+    cd ..
+}
+
+function init_bundle() {
+    local TEMPLATE_NAME="$1"
+    local BUNDLE_UUID="${2:-}"
+    local CONFIG_JSON="$3"
+    
+    # Extract project_name from JSON
+    local PROJECT_NAME=$(echo "$CONFIG_JSON" | grep -o '"project_name"[[:space:]]*:[[:space:]]*"[^"]*"' | cut -d'"' -f4)
+    
+    # Use 'cli' if available, otherwise fall back to 'databricks'
```

Comment:
why `cli`?

Reply:
`cli` might be a development build of `databricks` (it is on my system, at least :-o)